pods/covid-spain-chart/espana-covid-marzo.container.tsx

```diff
@@ -0,0 +1,12 @@
+import { EspanaCovidMarzoComponent } from "./espana-covid-marzo.component";
+import React from "react";
+
+interface Props {
+}
+
+export const EspanaCovidMarzoContainer: React.FC<Props> = (props: Props) => {
```

Comment:
We need to load here the name of the chart, the description, and where did we obtained this data source

Reply:
Would not be better to do so in "pages/espana-covid-marzo.tsx"?

Reply:
You are totally right, going to rename it